### PR TITLE
solving some problems with FromDocplex2IsingModel

### PR DESCRIPTION
**Pull request by alejomonbar**
While working on the tutorial, I found some problems with the converters class FromDocplex2IsingModel and I fixed them.

Additionally, I added some other things:

- a test for the new results attribute lowest_cost_bitstrings

- probabilities to the lowest_cost_bitstrings to see what's the probability for the best cases.

## Description

- Please include a summary of the changes and the related issue
- Please also include relevant context.
- List any dependencies that are required for this change, if any.
- **Fixes # (issue)**

## Checklist

[//]: <> (- [ ] My code follows the style guidelines of this project)

- [x] I have performed a self-review of my code.
- [x] I have commented my code and used numpy-style docstrings
- [ ] I have made corresponding updates to the documentation.
- [x] My changes generate no new warnings
- [x] I have added/updated tests to make sure bugfix/feature works.
- [x] New and existing unit tests pass locally with my changes.

[//]: <> (- [ ] Any dependent changes have been merged and published in downstream modules)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Name the new unit-tests that you have added along with this change.

### CODE REVIEW

**Review by vishal-ph — APPROVED** [review]

File: openqaoa/problems/converters.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 
 class FromDocplex2IsingModel:
-    def __init__(self, model, multipliers: float = None):
+    def __init__(self, model, multipliers: [float, list] = None):
```
Comment (shahidee44) [inline openqaoa/problems/converters.py:21]: For the multipliers here. Are the types float or list, or float and list?? If float or list, we should be using Union[float, list] here.

File: openqaoa/problems/converters.py [hunk 2]
```diff
@@ -327,7 +330,7 @@ def qubo_to_ising(n_variables, qubo_terms, qubo_weights):
 
         return QUBO(n_variables, ising_terms, ising_weights)
 
-    def get_models(self, multipliers: float = None):
+    def get_models(self, multipliers: [float, list] = None):
```
Comment (shahidee44) [inline openqaoa/problems/converters.py:333]: Same issue as the init function for this input parameter

**Review by shahidee44 — CHANGES_REQUESTED** [review]

File: openqaoa/problems/converters.py (hunk 1 above)
Comment (alejomonbar) [reply openqaoa/problems/converters.py]: Thanks @shahidee44, I updated it!

File: openqaoa/problems/converters.py (hunk 2 above)
Comment (alejomonbar) [reply openqaoa/problems/converters.py]: Thanks @shahidee44, I updated it!

**Review by shahidee44 — APPROVED** [review]

**Pull request merged**